UnityProject/Assets/Scripts/Clothing/RestraintOverlay.cs

```diff
@@ -98,9 +98,11 @@ IEnumerator UncuffCountDown(CancellationToken cancelToken)
 
 	bool CanUncuff()
 	{
-		if (!thisPlayerScript.playerHealth.serverPlayerConscious ||
+		PlayerHealth playerHealth = thisPlayerScript.playerHealth;
```

Comment:
Pretty sure PlayerHealth is null if you are a ghost. Is there anyway CanUncuff() might get called if you are a ghost? Maybe you should check that playerHealth exists and if it doesn't then return false. To be safe.